src/renderer/actions/tables.js

```diff
@@ -31,10 +31,10 @@ function shouldFetchTables (state, database) {
 
 
 function fetchTables (database, filter) {
```

Comment:
@s-kem I think you'll need to do the same fix for views, procedures etc.

Reply:
You're probably right. I added a commit for that. I'll test it tomorrow on my production databases which have some views and triggers across different databases. My personal dev machine just has the docker instances. I should probably add some views and functions to those so I can catch things like this.

Reply:
Have you tested it on production databases yet? :)

I'll test it out on my local dev instance.

Reply:
Looks alright to me. Please confirm if you have tested on production databases :)